salt/metalk8s/kubeadm/init/kubeconfig/admin.sls

```diff
@@ -0,0 +1,12 @@
+Create kubeconf file for admin:
+  metalk8s_kubeconfig.managed:
+    - name: /etc/kubernetes/admin.conf
+    - client_cert_info:
+        signing_private_key: /etc/kubernetes/pki/ca.key
+        signing_cert: /etc/kubernetes/pki/ca.crt
+        CN: kubernetes-admin
+        O: system:masters
```

Comment:
For me these value should be part of defaults.yaml

Reply:
@ramanduh can we resolve this one?